### ISSUE TITLE
On a mac, tsun throws errors

### ISSUE THREAD

**Issue by superandrew** [HerringtonDarkholme/typescript-repl#31]
I launched tsun, here's the output:

```
var name: string = 'Felipe';
Cannot find type definition file for 'jasmine'.
Cannot find type definition file for 'jasminewd2'.
Cannot find type definition file for 'node'.
Cannot find type definition file for 'q'.
Cannot find type definition file for 'selenium-webdriver'.
Cannot redeclare block-scoped variable 'name'.
```

No idea why.

**Comment by HerringtonDarkholme** [comment]
Probably because you launch tsun in a project file (with bad typing files).

tsun will automatically include typing files in your project. If you don't want this, you can launch it in a new empty directory.

**Comment by kbd** [comment]
I'm getting these errors too, yet I have the @types/... files added to `package.json`. I also tried installing `tsun` within the project and running the version in `./node_modules/.bin` instead of the one installed globally, but still get the errors.

Even if type files aren't found, this is something that would be good to print once, not after every line in the REPL.

**Comment by ORBAT** [comment]
I'm running into this same problem

- TSUN 0.3.8
- TypeScript 3.1.6
- NPM 6.4.1
- @types/node and @types/lodash installed as `devDependencies` and can be found in `node_modules`

When I run TSUN in the project with [this tsconfig](https://gist.github.com/ORBAT/74c8a5b85d3e7f7db19d060355be3332), I get the following errors:

```
> class Q { public a: number}
Cannot find type definition file for 'lodash'.
Cannot find type definition file for 'node'.
> Q
Cannot find type definition file for 'lodash'.
Cannot find type definition file for 'node'.
Cannot find name 'Q'.
```

When I run TSUN in a new directory, everything works as expected.

Now, @HerringtonDarkholme, as you said before it's about bad typing files and/or a misconfigured tsconfig, but you never really said how this should be fixed.

Any suggestions on what to do in my case?

**Comment by HerringtonDarkholme** [comment]
Please provide an example repository for bug report.

TypeScript now has a complicate resolution strategy. There is virtually no reliable solution to all projectes with subtly different configuration, other than start TSUN in a new directory.

**Comment by ORBAT** [comment]
@HerringtonDarkholme, https://github.com/ORBAT/tsun-problem-example

Thank you for taking the time to look at this

**Comment by ORBAT** [comment]
@HerringtonDarkholme, I pared down the example repo to just a `node_modules` directory. So if I start tsun in a directory with those specific modules in `node_modules` and try to eg. define a class, it fails.

Should I open a new issue for this? Anything I can do to help you debug it?

**Comment by HerringtonDarkholme** [comment]
@ORBAT You don't have a package.json or tsconfig.json or so for TS compiler to infer the project. Can you add them and then retry?

**Comment by ORBAT** [comment]
@HerringtonDarkholme, I re-added the tsconfig & package files into the repo. Same result, unfortunately

**Comment by HerringtonDarkholme** [comment]
I'm busy with my day jobs. I will not have much time to investigate this issue. But any investigation is welcome!

**Comment by HerringtonDarkholme** [comment]
I have solved the problem and published 0.4.0
